triggers/pixelbinEvents.js

```diff
@@ -0,0 +1,383 @@
+var hookID = "";
+const eventIds = [];
+
+const subscribeHook = async (z, bundle) => {
+	const { v4: uuidv4 } = require("uuid");
+	const zapier = require("zapier-platform-core");
+	zapier.tools.env.inject();
+	const eventIds = [];
+
+	const fetchEvents = {
+		url: `${process.env.BASE_URL}/service/platform/notification/v1.0/events`,
+		method: "GET",
+	};
+
+	try {
+		let response = await z.request(fetchEvents);
+
+		if (response.status === 200) {
+			const temp = [...response.data];
```

Comment:
remove this filter for each of them , just give all

Reply:
make it a dynamic field

Reply:
resolved